examples/distroless/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# ch-test-scope: standard
+# Distroless is a Google project providing slim images that contain runtime
+# dependencies only. https://github.com/GoogleContainerTools/distroless
+# The python3 image was chosen for ease of testing.
+FROM gcr.io/distroless/python3
+COPY . /distroless
```

Comment:
I suggest (a) copy only `hello.py` since that's all that's used, and (2) put it at the root directory.

Reply:
This was done to be consistent with the `hello` and `mpihello` tests. Is this something that you would like corrected in the other tests?

Reply:
Some of the others do put multiple files in. I would leave the others alone for now.